sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/DeviceNetworkSample.md

```diff
@@ -0,0 +1,15 @@
+# Network retrieve
+
+To get the Network name from a network identifier (IPv4/IPv6/NetworkCode), run the following code.
+
+```C# Snippet:APC_Sample_NetworkRetrievalTest
+string ApcGatewayId = "/subscriptions/abcdefgh/resourceGroups/dev-testing-eastus/providers/Microsoft.programmableconnectivity/gateways/apcg-eastus";
+Uri _endpoint = new Uri("https://your-endpoint-here.com");
+TokenCredential _credential = new DefaultAzureCredential();
+ProgrammableConnectivityClient baseClient = new ProgrammableConnectivityClient(_endpoint, _credential);
+var client = baseClient.GetDeviceNetworkClient();
```

Comment:
We discourage using `var` in samples for the results of method calls and other scenarios where the type is not clear to readers.

Reply:
I will go over all the tests and replace all `var`s with the actual types.